src/debugging/netSdk/NetSdkDebugHelper.ts

```diff
@@ -1,12 +1,18 @@
-import { commands } from "vscode";
+import * as path from "path";
+import { Task, Uri, commands, l10n, tasks } from "vscode";
+import { ext } from "../../extensionVariables";
 import { NetChooseBuildTypeContext, netContainerBuild } from "../../scaffolding/wizard/net/NetContainerBuild";
 import { AllNetContainerBuildOptions } from "../../scaffolding/wizard/net/NetSdkChooseBuildStep";
+import { getDefaultContainerName } from "../../tasks/TaskHelper";
+import { NetSdkRunTaskType, inferProjPath } from "../../tasks/netSdk/netSdkTaskUtils";
+import { PlatformOS } from "../../utils/platform";
 import { unresolveWorkspaceFolder } from "../../utils/resolveVariables";
-import { DockerDebugScaffoldContext } from "../DebugHelper";
+import { DockerDebugContext, DockerDebugScaffoldContext } from "../DebugHelper";
 import { DockerDebugConfiguration } from "../DockerDebugConfigurationProvider";
 import { NetCoreDebugHelper, NetCoreDebugScaffoldingOptions } from "../netcore/NetCoreDebugHelper";
 
-const NetSdkTaskFullSymbol = 'dotnet-container-sdk: debug';
+export const NetSdkProjPathMementoKey = 'netSdkProjPath';
```

Comment:
I'd prefer not to use a memento to store the target project, to avoid a user getting stuck debugging the wrong project; either a static variable or something in the extension context that will be cleared either when vscode closes or the extension is reloaded.

Reply:
the current implementation of memento is that we clear it every time one `f5`s. which means if you choose the wrong project, they can just f5 again and they will get prompt the option to choose a .csproj again. Or even if you choose the project you desire, next time you f5, you're going to get prompted again. 

If we want to change the user experience to only prompt to choose a `csproj` if user `f5`'s on a new vscode session, that's when we should use static variables. So it just depends on which user experience we think is best.

Reply:
In that case, we shouldn't actually need a memento at all if you define the task to run when you call execTask, since you can use that to provide the selected proejct.

Reply:
yeah i think you might be right

Reply:
got rid of momento